src/libpriv/rpmostree-origin.cxx

```diff
@@ -156,6 +156,9 @@ rpmostree_origin_parse_keyfile (GKeyFile         *origin,
   ret->cached_initramfs_args =
     g_key_file_get_string_list (ret->kf, "rpmostree", "initramfs-args", NULL, NULL);
 
+  // if (getenv ("RPMOSTREE_ORIGIN_TEST_ROUNDTRIP"))
```

Comment:
OK actually I'm going to re-impose draft because we need to merge with this line uncommented - I commented it out to run through CI.  But with this uncommented there is zero real runtime effect i.e. nil blast radius.